src/briefcase/windows_icon.py

```diff
@@ -0,0 +1,92 @@
+from ctypes import *
+from win32ctypes.pywin32 import win32api
+
+
+# Structures in ico file
+class IconHeader(Structure):
+    _fields_ = [
```

Comment:
At column 18: (F405) 'Structure' may be undefined, or defined from star imports: ctypes